packages/teleport/src/components/ResourceButtonAdd/ResourceButtonAdd.tsx

```diff
@@ -17,17 +17,17 @@ limitations under the License.
 import React from 'react';
 import { ButtonPrimary } from 'design';
 
-export default function ButtonAdd(props: Props) {
-  const { canCreate, isLeafCluster, onClick } = props;
+export default function ResourceButtonAdd(props: Props) {
+  const { canCreate, isLeafCluster, onClick, resource } = props;
   const disabled = isLeafCluster || !canCreate;
 
   let title = '';
   if (!canCreate) {
-    title = 'You do not have access to add an application';
+    title = `You do not have access to add ${resource}`;
```

Comment:
lets keep the `an` or `a` b/c the sentence flows better. We can another prop `beginsWithVowel` or something like that, so that when true we use `an`, otherwise `a`